src/app/products/page.tsx

```diff
@@ -0,0 +1,34 @@
+"use client";
+import Link from "next/link";
+import { Product } from "../types";
+import ItemCard from "../components/ItemCard";
+import { useProductsContext } from "../ProductsContextProvider";
+import { useProductIndexDispatchContext } from "../ProductIndexContextProvider";
+import { showStar } from "../utils";
+
+export default function Products() {
```

Comment:
It would be better to show products directly on the home page since right now home page is blank.